content/java/concepts/math-methods/terms/rint/rint.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: '.rint()'
+Description: 'Returns the closest whole double value to the value of the argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.rint()`** returns the closest whole double value to the value of the argument.
+
+## Syntax
+
+```pseudo
+Math.rint(num)
+```
+
+The `.rint()` method takes a double as an argument and returns a double.
```

Comment:
```suggestion
The `.rint()` method takes a double `num` parameter and returns a double.
```

In the context of describing the function/method definition in the "Syntax" section, let's use "parameter". Any other times when the function/method is used in an actual example, we should use "argument."